web/src/services/api.ts

```diff
@@ -66,8 +66,8 @@ export default class API {
       location.reload();
       return;
     }
-    if (response.status >= 400) {
-      throw new Error(await response.text());
+    if (response.statusText === 'Error in saveClip server') {
```

Comment:
I don't think we should replace the generic error handling here. Also the generic case would also cover this case wouldn't it?